python/cudf/cudf/core/column/datetime.py

```diff
@@ -109,6 +115,21 @@ class DatetimeColumn(column.ColumnBase):
         The validity mask
     """
 
+    # TODO: Timedelta columns support more operations than this, figure out why
```

Comment:
Ah thanks for pointing this out. This comment is no longer necessary, there is nothing to be done here. The original intent of this comment was to figure out why the Python `TimeDeltaColumn` has more supported operators than `DatetimeColumn`. As it turns out, it makes sense since you can multiply or divide time deltas, but you can't do that for absolute times. Additionally, the behavior of these two column types with respect to different types of scalars is also different. I plan to clarify and improve on this support in a subsequent PR, but there is no open question left here about why these two column types support different operators.